src/modules/simulation/gz_bridge/GZGimbal.cpp

```diff
@@ -228,7 +226,7 @@ void GZGimbal::publishDeviceAttitude()
 void GZGimbal::publishJointCommand(gz::transport::Node::Publisher &publisher, const float att_stp, const float rate_stp,
 				   float &last_stp, const float min_stp, const float max_stp, const float dt)
 {
-	gz::msgs::Double msg;
+	gz::msgs::Float msg;
```

Comment:
why did you change this? The underlying message type is double. This causes errors
```
Node::Publisher::Publish() Type mismatch.
	* Type advertised: gz.msgs.Double
	* Type published: gz.msgs.Float
```

Reply:
With gz::msgs::Double

```
PX4-Autopilot/src/modules/simulation/gz_bridge/GZGimbal.cpp:236:15: fatal error: implicit conversion increases floating-point precision: 'float' to 'double' [-Wdouble-promotion]
  236 |         msg.set_data(new_stp);
```

Didn't realize it caused downstream errors. Should probably be a cast instead then?